tools/gke-custom-org-policies/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
the script contains arrays and `[[` which are available in bash. The shebang line should be `#!/bin/bash`